projects/backend/utils/article.ts

```diff
@@ -0,0 +1,18 @@
+import { ArticleType, BlockElement, HTMLElement } from '../../common/src'
```

Comment:
Small point, but maybe this file can be named something more appropriate and moved into a package named more sensibly outside of utils?